keyboards/just60/keymaps/default/rules.mk

```diff
@@ -0,0 +1 @@
+BOOTLOADER = qmk-dfu
```

Comment:
This should probably be broken off into a different keymap, like `default_qmkdfu` or `thinxer` if this is intended for your own usage.

Setting this rule here will mean the firmware behaves differently when compiled from source vs. compiled in QMK Configurator, which doesn't use any keymap-level `config.h` or `rules.mk` files.

Reply:
I haven't got a look at the configurator yet and have no idea on how to support that (and the layout system as this keyboard supports many). I will remove this rule file.

Reply:
I can implement this in a separate PR. Don't concern yourself with it now; I'll tag you.